src/lib.rs

```diff
@@ -784,12 +784,12 @@ pub trait Surface {
     fn clear(&mut self, rect: Option<&Rect>, color: Option<(f32, f32, f32, f32)>, color_srgb: bool,
              depth: Option<f32>, stencil: Option<i32>);
 
-    /// Clears the color attachment of the target.
+    /// Clears the color attachment of the target. Converts the color to sRGB when applicable.
     fn clear_color(&mut self, red: f32, green: f32, blue: f32, alpha: f32) {
         self.clear(None, Some((red, green, blue, alpha)), false, None, None);
     }
 
-    /// Clears the color attachment of the target. The color is in sRGB format.
+    /// Clears the color attachment of the target. The color is expected to match the target's color space.
```

Comment:
This is quite confusing to readers who wonder why the `_srgb` function does not expect data in srgb format. Could you extend the rustdoc to say what the `_srgb` stands for? (the srgb extension).

Reply:
Should I remove this line from the commit or change the the second sentences of the rustdocs to:
clear_color - "The color is converted to sRGB when the target has sRGB format."
clear_color_srgb -  "This function disables `GL_FRAMEBUFFER_SRGB`, so the color should be in sRGB format."

AFAICT `clear_color_srgb` will work with RGB input on an RGB target, but that's obviously not the intention.

Reply:
The first sentence is fine: "Clears the color attachment of the target.". I was talking more about the stuff after the first sentence.